client-linuxapp/src/main.rs

```diff
@@ -425,10 +445,15 @@ async fn main() -> Result<()> {
         return Ok(());
     }
 
-    // Get owner info
-    let to1d = get_to1d_from_rv(dc.as_ref())
+    // Get rv entries
+    let rv_info = get_rv_info(dc.as_ref());
+
+    let client_list = get_client_list(rv_info.unwrap())
```

Comment:
Getting a full client list before going over it might make the processing of other RV entries a bit more complicated, since it detaches the looping over RV entries from the building of clients.

Reply:
What can we do as an alternative?

Reply:
In the next PR, which I plan to be focused exclusively on the retries, what I can do is loop over RV entries and, instead of passing a vector of `RendezvousInterpretedDirective` to `get_client_list`, just pass one entry to it (so `RendezvousInterpretedDirective` instead of `Vec<RendezvousInterpretedDirective>`.

What do you think? @puiterwijk

Reply:
That sounds decent.